test/unit/behaviours/age-restriction.test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behavior = require('../../../apps/right-to-rent-check/behaviours/age-restriction');
+const moment = require('moment');
+
+describe('apps/behaviours/age-restriction', () => {
+  it('exports a function', () => {
+    expect(Behavior).to.be.a('function');
+  });
+
+  class Base {
+    validate() {}
+    ValidationError() {}
+  }
+  let req;
+  let res;
+  let next;
+  let AgeRestriction;
+  let instance;
+  let ValidationError;
+
+  beforeEach(() => {
+    res = reqres.res();
+    req = reqres.req({
+      form: {
+        values: {}
+      }
+    });
+    next = sinon.stub();
+    AgeRestriction = Behavior(Base);
+    instance = new AgeRestriction();
+    sinon.stub(Base.prototype, 'validate');
+    ValidationError = sinon.stub(Base.prototype, 'ValidationError');
+  });
+  afterEach(() => {
+    Base.prototype.validate.restore();
+    Base.prototype.ValidationError.restore();
+  });
+
+  describe('validate()', () => {
+    it('calls parent method when tenant date of birth is greater than 17 years 11 months', () => {
+      const validDob = moment('1990-12-01');
+      req.form.values['tenant-dob'] = validDob;
+      instance.validate(req, res);
+
+      Base.prototype.validate.should.have.been.calledWith(req, res);
+    });
+    describe('when the tenant\'s date of birth is less than than 17 years 11 months', () => {
+      let invalidDob = moment().subtract(1, 'months');
+
+      it('calls next', () => {
+        req.form.values['tenant-dob'] = invalidDob;
+        instance.validate(req, res, next);
```

Comment:
These testa are all async and should be formatted as in the following example:
Otherwise none of these tests do what you think they are doing. 
```
it('my test', (done) => {
    instance.validate(req, res, (err) => {
        expect(err).to.not.exist;
        expect(req.thing).to.be.true;
        done();
    });
})

Also you don't need half these validate tests. Just two tests.

// when tenantDob is greater than ageRestriction, next is called with a validation error
// when tenantDob is less than ageRestriction, the parent validate method is called.